packages/i18n/src/locales/en_US.js

```diff
@@ -1059,12 +1059,6 @@ const translations = {
               example: 'db.collection.getIndexKeys()',
               parameters: {}
             },
-            getIndices: {
-              link: '',
-              description: '',
-              example: '',
-              parameters: {}
-            },
             getIndexSpecs: {
```

Comment:
Should we spare some time / create tickets to fill all the missing help entries?